Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -97,6 +97,18 @@ void ABenchmarkGymGameMode::BeginPlay()
 				Delegate.BindUObject(this, &ABenchmarkGymGameMode::GetPlayersConnected);
 				SpatialDriver->SpatialMetrics->SetCustomMetric(PlayersSpawnedMetricName, Delegate);
 			}
+
+			{
+				UserSuppliedMetric Delegate;
+				Delegate.BindUObject(this, &ABenchmarkGymGameMode::GetFPSValid);
```

Comment:
Driveby - but worth doing now - can we only register this delegate for all gamemode - and register the other 4 if you're authoritative?